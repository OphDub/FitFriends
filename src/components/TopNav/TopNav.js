import React from 'react';
import { NavLink } from 'react-router-dom';
import './TopNav.css';
import FontAwesomeIcon from '@fortawesome/react-fontawesome';
import faEdit from '@fortawesome/fontawesome-free-solid/faEdit';
import faUsers from '@fortawesome/fontawesome-free-solid/faUsers';
import faHistory from '@fortawesome/fontawesome-free-solid/faHistory';

export const TopNav = () => {
  return (
    <nav className="TopNav">
      <NavLink to="/workout">
        <button className="topnav-btn">
          <FontAwesomeIcon icon={faEdit} size="3x"/>
          <h4>
            Post Workout
          </h4>
        </button>
      </NavLink>
      <NavLink to="/team">
        <button className="topnav-btn">
          <FontAwesomeIcon icon={faUsers} size="3x"/>
          <h4>
            Team
          </h4>
        </button>
      </NavLink>
      <NavLink to="/history">
        <button className="topnav-btn">
          <FontAwesomeIcon icon={faHistory} size="3x"/>
          <h4>
            Past Workouts
          </h4>
        </button>
      </NavLink>
    </nav>
  );
};