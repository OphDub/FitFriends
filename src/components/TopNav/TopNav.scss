.TopNav {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  padding: 10px;
}

.TopNav a {
  width: 95px;
  height: 95px;
  text-decoration: none;
}

.topnav-btn {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  width: 100%;
  height: 100%;
  border: none;
  background: lightgrey;
  border-radius: 10px;
  outline: none;
}

.topnav-btn h4 {
  margin: 0px;
}

.topnav-btn:hover {
  color: white;
  background: black;
  cursor: pointer;
}