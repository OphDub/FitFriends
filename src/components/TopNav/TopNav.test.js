/* eslint-disable */
import React from 'react';
import { shallow } from 'enzyme';
import TopNav from './TopNav';

describe('TOPNAV', () => {
  it.skip('should match snapshot', () => {
    const renderedComponent = shallow(<TopNav />);

    expect(renderedComponent).toMatchSnapshot();
  })
})
