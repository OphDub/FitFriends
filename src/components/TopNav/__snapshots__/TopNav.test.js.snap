// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TOPNAV should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          to="/workout"
>
          <button
                    className="topnav-btn"
          >
                    <FontAwesomeIcon$1
                              border={false}
                              className=""
                              fixedWidth={false}
                              flip={null}
                              icon={
                                        Object {
                                                  "icon": Array [
                                                    576,
                                                    512,
                                                    Array [],
                                                    "f044",
                                                    "M402.6 83.2l90.2 90.2c3.8 3.8 3.8 10 0 13.8L274.4 405.6l-92.8 10.3c-12.4 1.4-22.9-9.1-21.5-21.5l10.3-92.8L388.8 83.2c3.8-3.8 10-3.8 13.8 0zm162-22.9l-48.8-48.8c-15.2-15.2-39.9-15.2-55.2 0l-35.4 35.4c-3.8 3.8-3.8 10 0 13.8l90.2 90.2c3.8 3.8 10 3.8 13.8 0l35.4-35.4c15.2-15.3 15.2-40 0-55.2zM384 346.2V448H64V128h229.8c3.2 0 6.2-1.3 8.5-3.5l40-40c7.6-7.6 2.2-20.5-8.5-20.5H48C21.5 64 0 85.5 0 112v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V306.2c0-10.7-12.9-16-20.5-8.5l-40 40c-2.2 2.3-3.5 5.3-3.5 8.5z",
                                                  ],
                                                  "iconName": "edit",
                                                  "prefix": "fas",
                                                }
                              }
                              listItem={false}
                              mask={null}
                              name=""
                              pull={null}
                              pulse={false}
                              rotation={null}
                              size="3x"
                              spin={false}
                              symbol={false}
                              transform={null}
                    />
                    <h4>
                              Post Workout
                    </h4>
          </button>
</NavLink>,
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          to="/history"
>
          <button
                    className="topnav-btn"
          >
                    <FontAwesomeIcon$1
                              border={false}
                              className=""
                              fixedWidth={false}
                              flip={null}
                              icon={
                                        Object {
                                                  "icon": Array [
                                                    512,
                                                    512,
                                                    Array [],
                                                    "f1da",
                                                    "M504 255.531c.253 136.64-111.18 248.372-247.82 248.468-59.015.042-113.223-20.53-155.822-54.911-11.077-8.94-11.905-25.541-1.839-35.607l11.267-11.267c8.609-8.609 22.353-9.551 31.891-1.984C173.062 425.135 212.781 440 256 440c101.705 0 184-82.311 184-184 0-101.705-82.311-184-184-184-48.814 0-93.149 18.969-126.068 49.932l50.754 50.754c10.08 10.08 2.941 27.314-11.313 27.314H24c-8.837 0-16-7.163-16-16V38.627c0-14.254 17.234-21.393 27.314-11.314l49.372 49.372C129.209 34.136 189.552 8 256 8c136.81 0 247.747 110.78 248 247.531zm-180.912 78.784l9.823-12.63c8.138-10.463 6.253-25.542-4.21-33.679L288 256.349V152c0-13.255-10.745-24-24-24h-16c-13.255 0-24 10.745-24 24v135.651l65.409 50.874c10.463 8.137 25.541 6.253 33.679-4.21z",
                                                  ],
                                                  "iconName": "history",
                                                  "prefix": "fas",
                                                }
                              }
                              listItem={false}
                              mask={null}
                              name=""
                              pull={null}
                              pulse={false}
                              rotation={null}
                              size="3x"
                              spin={false}
                              symbol={false}
                              transform={null}
                    />
                    <h4>
                              Past Workouts
                    </h4>
          </button>
</NavLink>,
      ],
      "className": "TopNav",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": <button
            className="topnav-btn"
>
            <FontAwesomeIcon$1
                        border={false}
                        className=""
                        fixedWidth={false}
                        flip={null}
                        icon={
                                    Object {
                                                "icon": Array [
                                                  576,
                                                  512,
                                                  Array [],
                                                  "f044",
                                                  "M402.6 83.2l90.2 90.2c3.8 3.8 3.8 10 0 13.8L274.4 405.6l-92.8 10.3c-12.4 1.4-22.9-9.1-21.5-21.5l10.3-92.8L388.8 83.2c3.8-3.8 10-3.8 13.8 0zm162-22.9l-48.8-48.8c-15.2-15.2-39.9-15.2-55.2 0l-35.4 35.4c-3.8 3.8-3.8 10 0 13.8l90.2 90.2c3.8 3.8 10 3.8 13.8 0l35.4-35.4c15.2-15.3 15.2-40 0-55.2zM384 346.2V448H64V128h229.8c3.2 0 6.2-1.3 8.5-3.5l40-40c7.6-7.6 2.2-20.5-8.5-20.5H48C21.5 64 0 85.5 0 112v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V306.2c0-10.7-12.9-16-20.5-8.5l-40 40c-2.2 2.3-3.5 5.3-3.5 8.5z",
                                                ],
                                                "iconName": "edit",
                                                "prefix": "fas",
                                              }
                        }
                        listItem={false}
                        mask={null}
                        name=""
                        pull={null}
                        pulse={false}
                        rotation={null}
                        size="3x"
                        spin={false}
                        symbol={false}
                        transform={null}
            />
            <h4>
                        Post Workout
            </h4>
</button>,
          "to": "/workout",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <FontAwesomeIcon$1
                border={false}
                className=""
                fixedWidth={false}
                flip={null}
                icon={
                                Object {
                                                "icon": Array [
                                                  576,
                                                  512,
                                                  Array [],
                                                  "f044",
                                                  "M402.6 83.2l90.2 90.2c3.8 3.8 3.8 10 0 13.8L274.4 405.6l-92.8 10.3c-12.4 1.4-22.9-9.1-21.5-21.5l10.3-92.8L388.8 83.2c3.8-3.8 10-3.8 13.8 0zm162-22.9l-48.8-48.8c-15.2-15.2-39.9-15.2-55.2 0l-35.4 35.4c-3.8 3.8-3.8 10 0 13.8l90.2 90.2c3.8 3.8 10 3.8 13.8 0l35.4-35.4c15.2-15.3 15.2-40 0-55.2zM384 346.2V448H64V128h229.8c3.2 0 6.2-1.3 8.5-3.5l40-40c7.6-7.6 2.2-20.5-8.5-20.5H48C21.5 64 0 85.5 0 112v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V306.2c0-10.7-12.9-16-20.5-8.5l-40 40c-2.2 2.3-3.5 5.3-3.5 8.5z",
                                                ],
                                                "iconName": "edit",
                                                "prefix": "fas",
                                              }
                }
                listItem={false}
                mask={null}
                name=""
                pull={null}
                pulse={false}
                rotation={null}
                size="3x"
                spin={false}
                symbol={false}
                transform={null}
/>,
              <h4>
                Post Workout
</h4>,
            ],
            "className": "topnav-btn",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "border": false,
                "className": "",
                "fixedWidth": false,
                "flip": null,
                "icon": Object {
                  "icon": Array [
                    576,
                    512,
                    Array [],
                    "f044",
                    "M402.6 83.2l90.2 90.2c3.8 3.8 3.8 10 0 13.8L274.4 405.6l-92.8 10.3c-12.4 1.4-22.9-9.1-21.5-21.5l10.3-92.8L388.8 83.2c3.8-3.8 10-3.8 13.8 0zm162-22.9l-48.8-48.8c-15.2-15.2-39.9-15.2-55.2 0l-35.4 35.4c-3.8 3.8-3.8 10 0 13.8l90.2 90.2c3.8 3.8 10 3.8 13.8 0l35.4-35.4c15.2-15.3 15.2-40 0-55.2zM384 346.2V448H64V128h229.8c3.2 0 6.2-1.3 8.5-3.5l40-40c7.6-7.6 2.2-20.5-8.5-20.5H48C21.5 64 0 85.5 0 112v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V306.2c0-10.7-12.9-16-20.5-8.5l-40 40c-2.2 2.3-3.5 5.3-3.5 8.5z",
                  ],
                  "iconName": "edit",
                  "prefix": "fas",
                },
                "listItem": false,
                "mask": null,
                "name": "",
                "pull": null,
                "pulse": false,
                "rotation": null,
                "size": "3x",
                "spin": false,
                "symbol": false,
                "transform": null,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Post Workout",
              },
              "ref": null,
              "rendered": "Post Workout",
              "type": "h4",
            },
          ],
          "type": "button",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": <button
            className="topnav-btn"
>
            <FontAwesomeIcon$1
                        border={false}
                        className=""
                        fixedWidth={false}
                        flip={null}
                        icon={
                                    Object {
                                                "icon": Array [
                                                  512,
                                                  512,
                                                  Array [],
                                                  "f1da",
                                                  "M504 255.531c.253 136.64-111.18 248.372-247.82 248.468-59.015.042-113.223-20.53-155.822-54.911-11.077-8.94-11.905-25.541-1.839-35.607l11.267-11.267c8.609-8.609 22.353-9.551 31.891-1.984C173.062 425.135 212.781 440 256 440c101.705 0 184-82.311 184-184 0-101.705-82.311-184-184-184-48.814 0-93.149 18.969-126.068 49.932l50.754 50.754c10.08 10.08 2.941 27.314-11.313 27.314H24c-8.837 0-16-7.163-16-16V38.627c0-14.254 17.234-21.393 27.314-11.314l49.372 49.372C129.209 34.136 189.552 8 256 8c136.81 0 247.747 110.78 248 247.531zm-180.912 78.784l9.823-12.63c8.138-10.463 6.253-25.542-4.21-33.679L288 256.349V152c0-13.255-10.745-24-24-24h-16c-13.255 0-24 10.745-24 24v135.651l65.409 50.874c10.463 8.137 25.541 6.253 33.679-4.21z",
                                                ],
                                                "iconName": "history",
                                                "prefix": "fas",
                                              }
                        }
                        listItem={false}
                        mask={null}
                        name=""
                        pull={null}
                        pulse={false}
                        rotation={null}
                        size="3x"
                        spin={false}
                        symbol={false}
                        transform={null}
            />
            <h4>
                        Past Workouts
            </h4>
</button>,
          "to": "/history",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <FontAwesomeIcon$1
                border={false}
                className=""
                fixedWidth={false}
                flip={null}
                icon={
                                Object {
                                                "icon": Array [
                                                  512,
                                                  512,
                                                  Array [],
                                                  "f1da",
                                                  "M504 255.531c.253 136.64-111.18 248.372-247.82 248.468-59.015.042-113.223-20.53-155.822-54.911-11.077-8.94-11.905-25.541-1.839-35.607l11.267-11.267c8.609-8.609 22.353-9.551 31.891-1.984C173.062 425.135 212.781 440 256 440c101.705 0 184-82.311 184-184 0-101.705-82.311-184-184-184-48.814 0-93.149 18.969-126.068 49.932l50.754 50.754c10.08 10.08 2.941 27.314-11.313 27.314H24c-8.837 0-16-7.163-16-16V38.627c0-14.254 17.234-21.393 27.314-11.314l49.372 49.372C129.209 34.136 189.552 8 256 8c136.81 0 247.747 110.78 248 247.531zm-180.912 78.784l9.823-12.63c8.138-10.463 6.253-25.542-4.21-33.679L288 256.349V152c0-13.255-10.745-24-24-24h-16c-13.255 0-24 10.745-24 24v135.651l65.409 50.874c10.463 8.137 25.541 6.253 33.679-4.21z",
                                                ],
                                                "iconName": "history",
                                                "prefix": "fas",
                                              }
                }
                listItem={false}
                mask={null}
                name=""
                pull={null}
                pulse={false}
                rotation={null}
                size="3x"
                spin={false}
                symbol={false}
                transform={null}
/>,
              <h4>
                Past Workouts
</h4>,
            ],
            "className": "topnav-btn",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "border": false,
                "className": "",
                "fixedWidth": false,
                "flip": null,
                "icon": Object {
                  "icon": Array [
                    512,
                    512,
                    Array [],
                    "f1da",
                    "M504 255.531c.253 136.64-111.18 248.372-247.82 248.468-59.015.042-113.223-20.53-155.822-54.911-11.077-8.94-11.905-25.541-1.839-35.607l11.267-11.267c8.609-8.609 22.353-9.551 31.891-1.984C173.062 425.135 212.781 440 256 440c101.705 0 184-82.311 184-184 0-101.705-82.311-184-184-184-48.814 0-93.149 18.969-126.068 49.932l50.754 50.754c10.08 10.08 2.941 27.314-11.313 27.314H24c-8.837 0-16-7.163-16-16V38.627c0-14.254 17.234-21.393 27.314-11.314l49.372 49.372C129.209 34.136 189.552 8 256 8c136.81 0 247.747 110.78 248 247.531zm-180.912 78.784l9.823-12.63c8.138-10.463 6.253-25.542-4.21-33.679L288 256.349V152c0-13.255-10.745-24-24-24h-16c-13.255 0-24 10.745-24 24v135.651l65.409 50.874c10.463 8.137 25.541 6.253 33.679-4.21z",
                  ],
                  "iconName": "history",
                  "prefix": "fas",
                },
                "listItem": false,
                "mask": null,
                "name": "",
                "pull": null,
                "pulse": false,
                "rotation": null,
                "size": "3x",
                "spin": false,
                "symbol": false,
                "transform": null,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Past Workouts",
              },
              "ref": null,
              "rendered": "Past Workouts",
              "type": "h4",
            },
          ],
          "type": "button",
        },
        "type": [Function],
      },
    ],
    "type": "nav",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            to="/workout"
>
            <button
                        className="topnav-btn"
            >
                        <FontAwesomeIcon$1
                                    border={false}
                                    className=""
                                    fixedWidth={false}
                                    flip={null}
                                    icon={
                                                Object {
                                                            "icon": Array [
                                                              576,
                                                              512,
                                                              Array [],
                                                              "f044",
                                                              "M402.6 83.2l90.2 90.2c3.8 3.8 3.8 10 0 13.8L274.4 405.6l-92.8 10.3c-12.4 1.4-22.9-9.1-21.5-21.5l10.3-92.8L388.8 83.2c3.8-3.8 10-3.8 13.8 0zm162-22.9l-48.8-48.8c-15.2-15.2-39.9-15.2-55.2 0l-35.4 35.4c-3.8 3.8-3.8 10 0 13.8l90.2 90.2c3.8 3.8 10 3.8 13.8 0l35.4-35.4c15.2-15.3 15.2-40 0-55.2zM384 346.2V448H64V128h229.8c3.2 0 6.2-1.3 8.5-3.5l40-40c7.6-7.6 2.2-20.5-8.5-20.5H48C21.5 64 0 85.5 0 112v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V306.2c0-10.7-12.9-16-20.5-8.5l-40 40c-2.2 2.3-3.5 5.3-3.5 8.5z",
                                                            ],
                                                            "iconName": "edit",
                                                            "prefix": "fas",
                                                          }
                                    }
                                    listItem={false}
                                    mask={null}
                                    name=""
                                    pull={null}
                                    pulse={false}
                                    rotation={null}
                                    size="3x"
                                    spin={false}
                                    symbol={false}
                                    transform={null}
                        />
                        <h4>
                                    Post Workout
                        </h4>
            </button>
</NavLink>,
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            to="/history"
>
            <button
                        className="topnav-btn"
            >
                        <FontAwesomeIcon$1
                                    border={false}
                                    className=""
                                    fixedWidth={false}
                                    flip={null}
                                    icon={
                                                Object {
                                                            "icon": Array [
                                                              512,
                                                              512,
                                                              Array [],
                                                              "f1da",
                                                              "M504 255.531c.253 136.64-111.18 248.372-247.82 248.468-59.015.042-113.223-20.53-155.822-54.911-11.077-8.94-11.905-25.541-1.839-35.607l11.267-11.267c8.609-8.609 22.353-9.551 31.891-1.984C173.062 425.135 212.781 440 256 440c101.705 0 184-82.311 184-184 0-101.705-82.311-184-184-184-48.814 0-93.149 18.969-126.068 49.932l50.754 50.754c10.08 10.08 2.941 27.314-11.313 27.314H24c-8.837 0-16-7.163-16-16V38.627c0-14.254 17.234-21.393 27.314-11.314l49.372 49.372C129.209 34.136 189.552 8 256 8c136.81 0 247.747 110.78 248 247.531zm-180.912 78.784l9.823-12.63c8.138-10.463 6.253-25.542-4.21-33.679L288 256.349V152c0-13.255-10.745-24-24-24h-16c-13.255 0-24 10.745-24 24v135.651l65.409 50.874c10.463 8.137 25.541 6.253 33.679-4.21z",
                                                            ],
                                                            "iconName": "history",
                                                            "prefix": "fas",
                                                          }
                                    }
                                    listItem={false}
                                    mask={null}
                                    name=""
                                    pull={null}
                                    pulse={false}
                                    rotation={null}
                                    size="3x"
                                    spin={false}
                                    symbol={false}
                                    transform={null}
                        />
                        <h4>
                                    Past Workouts
                        </h4>
            </button>
</NavLink>,
        ],
        "className": "TopNav",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": <button
              className="topnav-btn"
>
              <FontAwesomeIcon$1
                            border={false}
                            className=""
                            fixedWidth={false}
                            flip={null}
                            icon={
                                          Object {
                                                        "icon": Array [
                                                          576,
                                                          512,
                                                          Array [],
                                                          "f044",
                                                          "M402.6 83.2l90.2 90.2c3.8 3.8 3.8 10 0 13.8L274.4 405.6l-92.8 10.3c-12.4 1.4-22.9-9.1-21.5-21.5l10.3-92.8L388.8 83.2c3.8-3.8 10-3.8 13.8 0zm162-22.9l-48.8-48.8c-15.2-15.2-39.9-15.2-55.2 0l-35.4 35.4c-3.8 3.8-3.8 10 0 13.8l90.2 90.2c3.8 3.8 10 3.8 13.8 0l35.4-35.4c15.2-15.3 15.2-40 0-55.2zM384 346.2V448H64V128h229.8c3.2 0 6.2-1.3 8.5-3.5l40-40c7.6-7.6 2.2-20.5-8.5-20.5H48C21.5 64 0 85.5 0 112v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V306.2c0-10.7-12.9-16-20.5-8.5l-40 40c-2.2 2.3-3.5 5.3-3.5 8.5z",
                                                        ],
                                                        "iconName": "edit",
                                                        "prefix": "fas",
                                                      }
                            }
                            listItem={false}
                            mask={null}
                            name=""
                            pull={null}
                            pulse={false}
                            rotation={null}
                            size="3x"
                            spin={false}
                            symbol={false}
                            transform={null}
              />
              <h4>
                            Post Workout
              </h4>
</button>,
            "to": "/workout",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <FontAwesomeIcon$1
                  border={false}
                  className=""
                  fixedWidth={false}
                  flip={null}
                  icon={
                                    Object {
                                                      "icon": Array [
                                                        576,
                                                        512,
                                                        Array [],
                                                        "f044",
                                                        "M402.6 83.2l90.2 90.2c3.8 3.8 3.8 10 0 13.8L274.4 405.6l-92.8 10.3c-12.4 1.4-22.9-9.1-21.5-21.5l10.3-92.8L388.8 83.2c3.8-3.8 10-3.8 13.8 0zm162-22.9l-48.8-48.8c-15.2-15.2-39.9-15.2-55.2 0l-35.4 35.4c-3.8 3.8-3.8 10 0 13.8l90.2 90.2c3.8 3.8 10 3.8 13.8 0l35.4-35.4c15.2-15.3 15.2-40 0-55.2zM384 346.2V448H64V128h229.8c3.2 0 6.2-1.3 8.5-3.5l40-40c7.6-7.6 2.2-20.5-8.5-20.5H48C21.5 64 0 85.5 0 112v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V306.2c0-10.7-12.9-16-20.5-8.5l-40 40c-2.2 2.3-3.5 5.3-3.5 8.5z",
                                                      ],
                                                      "iconName": "edit",
                                                      "prefix": "fas",
                                                    }
                  }
                  listItem={false}
                  mask={null}
                  name=""
                  pull={null}
                  pulse={false}
                  rotation={null}
                  size="3x"
                  spin={false}
                  symbol={false}
                  transform={null}
/>,
                <h4>
                  Post Workout
</h4>,
              ],
              "className": "topnav-btn",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "border": false,
                  "className": "",
                  "fixedWidth": false,
                  "flip": null,
                  "icon": Object {
                    "icon": Array [
                      576,
                      512,
                      Array [],
                      "f044",
                      "M402.6 83.2l90.2 90.2c3.8 3.8 3.8 10 0 13.8L274.4 405.6l-92.8 10.3c-12.4 1.4-22.9-9.1-21.5-21.5l10.3-92.8L388.8 83.2c3.8-3.8 10-3.8 13.8 0zm162-22.9l-48.8-48.8c-15.2-15.2-39.9-15.2-55.2 0l-35.4 35.4c-3.8 3.8-3.8 10 0 13.8l90.2 90.2c3.8 3.8 10 3.8 13.8 0l35.4-35.4c15.2-15.3 15.2-40 0-55.2zM384 346.2V448H64V128h229.8c3.2 0 6.2-1.3 8.5-3.5l40-40c7.6-7.6 2.2-20.5-8.5-20.5H48C21.5 64 0 85.5 0 112v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V306.2c0-10.7-12.9-16-20.5-8.5l-40 40c-2.2 2.3-3.5 5.3-3.5 8.5z",
                    ],
                    "iconName": "edit",
                    "prefix": "fas",
                  },
                  "listItem": false,
                  "mask": null,
                  "name": "",
                  "pull": null,
                  "pulse": false,
                  "rotation": null,
                  "size": "3x",
                  "spin": false,
                  "symbol": false,
                  "transform": null,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Post Workout",
                },
                "ref": null,
                "rendered": "Post Workout",
                "type": "h4",
              },
            ],
            "type": "button",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": <button
              className="topnav-btn"
>
              <FontAwesomeIcon$1
                            border={false}
                            className=""
                            fixedWidth={false}
                            flip={null}
                            icon={
                                          Object {
                                                        "icon": Array [
                                                          512,
                                                          512,
                                                          Array [],
                                                          "f1da",
                                                          "M504 255.531c.253 136.64-111.18 248.372-247.82 248.468-59.015.042-113.223-20.53-155.822-54.911-11.077-8.94-11.905-25.541-1.839-35.607l11.267-11.267c8.609-8.609 22.353-9.551 31.891-1.984C173.062 425.135 212.781 440 256 440c101.705 0 184-82.311 184-184 0-101.705-82.311-184-184-184-48.814 0-93.149 18.969-126.068 49.932l50.754 50.754c10.08 10.08 2.941 27.314-11.313 27.314H24c-8.837 0-16-7.163-16-16V38.627c0-14.254 17.234-21.393 27.314-11.314l49.372 49.372C129.209 34.136 189.552 8 256 8c136.81 0 247.747 110.78 248 247.531zm-180.912 78.784l9.823-12.63c8.138-10.463 6.253-25.542-4.21-33.679L288 256.349V152c0-13.255-10.745-24-24-24h-16c-13.255 0-24 10.745-24 24v135.651l65.409 50.874c10.463 8.137 25.541 6.253 33.679-4.21z",
                                                        ],
                                                        "iconName": "history",
                                                        "prefix": "fas",
                                                      }
                            }
                            listItem={false}
                            mask={null}
                            name=""
                            pull={null}
                            pulse={false}
                            rotation={null}
                            size="3x"
                            spin={false}
                            symbol={false}
                            transform={null}
              />
              <h4>
                            Past Workouts
              </h4>
</button>,
            "to": "/history",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <FontAwesomeIcon$1
                  border={false}
                  className=""
                  fixedWidth={false}
                  flip={null}
                  icon={
                                    Object {
                                                      "icon": Array [
                                                        512,
                                                        512,
                                                        Array [],
                                                        "f1da",
                                                        "M504 255.531c.253 136.64-111.18 248.372-247.82 248.468-59.015.042-113.223-20.53-155.822-54.911-11.077-8.94-11.905-25.541-1.839-35.607l11.267-11.267c8.609-8.609 22.353-9.551 31.891-1.984C173.062 425.135 212.781 440 256 440c101.705 0 184-82.311 184-184 0-101.705-82.311-184-184-184-48.814 0-93.149 18.969-126.068 49.932l50.754 50.754c10.08 10.08 2.941 27.314-11.313 27.314H24c-8.837 0-16-7.163-16-16V38.627c0-14.254 17.234-21.393 27.314-11.314l49.372 49.372C129.209 34.136 189.552 8 256 8c136.81 0 247.747 110.78 248 247.531zm-180.912 78.784l9.823-12.63c8.138-10.463 6.253-25.542-4.21-33.679L288 256.349V152c0-13.255-10.745-24-24-24h-16c-13.255 0-24 10.745-24 24v135.651l65.409 50.874c10.463 8.137 25.541 6.253 33.679-4.21z",
                                                      ],
                                                      "iconName": "history",
                                                      "prefix": "fas",
                                                    }
                  }
                  listItem={false}
                  mask={null}
                  name=""
                  pull={null}
                  pulse={false}
                  rotation={null}
                  size="3x"
                  spin={false}
                  symbol={false}
                  transform={null}
/>,
                <h4>
                  Past Workouts
</h4>,
              ],
              "className": "topnav-btn",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "border": false,
                  "className": "",
                  "fixedWidth": false,
                  "flip": null,
                  "icon": Object {
                    "icon": Array [
                      512,
                      512,
                      Array [],
                      "f1da",
                      "M504 255.531c.253 136.64-111.18 248.372-247.82 248.468-59.015.042-113.223-20.53-155.822-54.911-11.077-8.94-11.905-25.541-1.839-35.607l11.267-11.267c8.609-8.609 22.353-9.551 31.891-1.984C173.062 425.135 212.781 440 256 440c101.705 0 184-82.311 184-184 0-101.705-82.311-184-184-184-48.814 0-93.149 18.969-126.068 49.932l50.754 50.754c10.08 10.08 2.941 27.314-11.313 27.314H24c-8.837 0-16-7.163-16-16V38.627c0-14.254 17.234-21.393 27.314-11.314l49.372 49.372C129.209 34.136 189.552 8 256 8c136.81 0 247.747 110.78 248 247.531zm-180.912 78.784l9.823-12.63c8.138-10.463 6.253-25.542-4.21-33.679L288 256.349V152c0-13.255-10.745-24-24-24h-16c-13.255 0-24 10.745-24 24v135.651l65.409 50.874c10.463 8.137 25.541 6.253 33.679-4.21z",
                    ],
                    "iconName": "history",
                    "prefix": "fas",
                  },
                  "listItem": false,
                  "mask": null,
                  "name": "",
                  "pull": null,
                  "pulse": false,
                  "rotation": null,
                  "size": "3x",
                  "spin": false,
                  "symbol": false,
                  "transform": null,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Past Workouts",
                },
                "ref": null,
                "rendered": "Past Workouts",
                "type": "h4",
              },
            ],
            "type": "button",
          },
          "type": [Function],
        },
      ],
      "type": "nav",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
